ntbs-service/Authentication/DummyAuthHandler.cs

```diff
@@ -9,6 +9,7 @@ namespace ntbs_service.Authentication {
     public class DummyAuthHandler : AuthenticationHandler<AuthenticationSchemeOptions>
     {
         public static readonly string Name = "DummyAuth";
+        public static readonly string Email = "DummyEmail";
```

Comment:
should we use email format, so sth like `"developer@ntbs.phe.com"`?
Also a bit confusing, it makes it seem like the constant above is also the developer name, whilst its actually the auth scheme's name - let's add the "Developer" string into constants too, or have the `Email` inlined :)